website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -102,10 +102,13 @@ metrics:
 | time_grains | One or more "grains" at which the metric can be evaluated   | [day, week, month]              | yes       |
 | dimensions  | A list of dimensions to group or filter the metric by       | [plan, country]                 | no        |
 | filters     | A list of filters to apply before calculating the metric    | See below                       | no        |
+| operator    | Defines the type of filter to use                           | '!='                            | yes       |
```

Comment:
```suggestion
```

I wouldn't include this here because it's a child of the filters object - if we were going to do that, we'd want to bring through all three of field, operator and value. Then we'd need to clarify that they're all required if one of them is provided, but it would get a bit messy. 

I'd rather describe it in the Filters block below as you've done